pkg/controller/elasticsearch/stackmon/container.go

```diff
@@ -0,0 +1,176 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package stackmon
+
+import (
+	"errors"
+	"path/filepath"
+	"strings"
+
+	esv1 "github.com/elastic/cloud-on-k8s/pkg/apis/elasticsearch/v1"
+	"github.com/elastic/cloud-on-k8s/pkg/controller/common/container"
+	"github.com/elastic/cloud-on-k8s/pkg/controller/common/defaults"
+	"github.com/elastic/cloud-on-k8s/pkg/controller/common/volume"
+	esvolume "github.com/elastic/cloud-on-k8s/pkg/controller/elasticsearch/volume"
+	corev1 "k8s.io/api/core/v1"
+)
+
+const (
+	esLogStyleEnvVarKey   = "ES_LOG_STYLE"
+	esLogStyleEnvVarValue = "file"
```

Comment:
let's add a comment about how this is a configuration option for the ES container (could be hard to understand what's beat config vs. what's ES config)